### PR TITLE
[WIP] Added Labs Page

### LINKED ISSUE #2947: As a user, I want gitcoin.co/labs so i can see all of the great works being done in Labs
### User Story

[comment]: # (As a <user type>, I want to <task> so that <goal>.)
As a user, I want gitcoin.co/labs so i can see all of the great works being done in Labs
### Description

[comment]: # (Feature or Bug? i.e Type: Bug)
*Type*:
feature 

### Current Behavior
[comment]: # (Describe what actually happened.)
no labs homepage 
### Expected Behavior
[comment]: # (Describe what you expected to happen.)

1. take the existing PR https://github.com/gitcoinco/web/pull/2801 and extend it such that
2. gitcoin.co/labs shows this design

3. 'Latest Research' must be database - driven.  It must be populated from a list of `Labs_Research` objects (each of which have an image, title, description, and link).  This database table must be available in teh django admin.
4. Tools must be populated from the `tools` table in reverse chronological order.  The tools DB object already exists.

For team page,  Austin info is posted at https://github.com/gitcoinco/web/issues/2864

Links at bottom
- Github -> https://github.com/gitcoinco/labs/
- Slack -> gitcoin.co/slack
- contaact -> founders@gitcoin.co

### Definition of Done
[comment]: # (Any other information that would be useful, bullets are helpful.)

- design must be responsive
- design must be performant
- pls allow for 1-2 revisions of the design.

### Additional Information
[comment]: # (Any other information that would be useful, content, screenshots, etc.)
designs: https://github.com/gitcoinco/web/issues/2821#issuecomment-441715027
design ticket: https://github.com/gitcoinco/web/issues/2821#issuecomment-440731513

### PR DESCRIPTION
**Pull request by aashutoshrathi**
##### Description

Create a Lab_Research Model and make it available to django admin
Also, I'll fetch tools from tools table.

##### Checklist

- [ ] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://docs.gitcoin.co/mk_contributors/#step-4-commit)

##### Refers/Fixes
Fixes #2947

### CODE REVIEW

File: app/retail/templates/shared/top_nav.html [hunk 1]
```diff
@@ -30,8 +30,12 @@
     <img src="{% static "v2/images/top-bar/kudos-logo.svg" %}" height="22" alt="" class="d-md-block d-none">
     <img src="{% static "v2/images/top-bar/kudos-symbol.svg" %}" height="22" alt="" class="d-md-none">
   </a>
-  <a class="{% if request.path|matches:'^\/((grants.*))$' %}selected{% endif %}" href="/grants">
+  <a class="{% if request.path|matches:'^\/((grants.*))$' %}selected{% endif %}" href="{% url 'labs' %}">
```
Comment (owocki) [inline app/retail/templates/shared/top_nav.html]: this seems wrong

File: app/kudos/migrations/0008_auto_20181129_2213.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('kudos', '0007_merge_20181114_2349'),
+    ]
+
+    operations = [
```
Comment (owocki) [inline app/kudos/migrations/0008_auto_20181129_2213.py]: did you mean to commit this migration

File: app/dashboard/views.py [hunk 3]
```diff
@@ -1358,6 +1358,10 @@ def apitos(request):
     return TemplateResponse(request, 'legal/privacy.html', {})
 
 
+def labs(request):
+    return TemplateResponse(request, 'legal/privacy.html', {})
```
Comment (owocki) [inline app/dashboard/views.py]: legal/privacy.html?

Comment (aashutoshrathi) [reply app/dashboard/views.py]: Ahh.. that was just temporary, I'm creating the template for it.

File: app/kudos/migrations/0008_auto_20181129_2213.py (hunk 2 above)
Comment (aashutoshrathi) [reply app/kudos/migrations/0008_auto_20181129_2213.py]: No, I'll reset the migrations

File: app/retail/templates/shared/top_nav.html (hunk 1 above)
Comment (aashutoshrathi) [reply app/retail/templates/shared/top_nav.html]: Yeah, changing this to `grants`.

**Comment by owocki** [comment]
okie dokie @aashutoshrathi let me know when ready for review.. ill stop pestering until then

**Comment by aashutoshrathi** [comment]
Moved to #3048

**Pull request closed without merging**